cmd/angryjet/main.go

```diff
@@ -209,3 +219,27 @@ func GenerateProviderConfigUsageList(filename, header string, p *packages.Packag
 
 	return errors.Wrap(err, "cannot write provider config usage list methods")
 }
+
+// GenerateReferences generates reference resolver calls.
+func GenerateReferences(filename, header string, p *packages.Package) error {
+	receiver := "mg"
+	comm := comments.In(p)
+
+	methods := method.Set{
+		"ResolveReferences": method.NewResolveReferences(types.NewTraverser(comm), receiver, ClientImport, ReferenceImport),
```

Comment:
I don't love this structure since it'd mean that if we add another field/named processor, then we'd have to run a separate traverse operation. I thought of having `cmd/hostess/main.go` that is optimized for traversing but didn't want to do it in this PR. Maybe in the next traverse addition, like webhook generation for `+immutable` tag.